web/src/beta/utils/file.ts

```
@@ -40,3 +41,30 @@
 
   return file;
 }
+
+const CHUNK_SIZE = 16 * 1024 * 1024; // 16MB
+
+export const uploadFile = async (file: File, teamId: string, axios: any) => {
+  const totalChunks = Math.ceil(file.size / CHUNK_SIZE);
+  const fileId = uuidv4();
+
+  for (let chunkNum = 0; chunkNum < totalChunks; chunkNum++) {
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)